tests/deleteArticle.test.js

```diff
@@ -2,7 +2,6 @@ import chai from 'chai';
 import chaiHttp from 'chai-http';
```

Comment:
I made changes he because one of @MCFrank16 middleware was not using the `errorSender` helper yet I needed to use it. So after modified it, I had also to modify the way he was expecting the error object to look like

Reply:
@nkpremices I appreciate all these comments you've dropped.

Reply:
Thank you @MemunaHaruna